### ISSUE TITLE
Lists do not contain only <li> elements and script supporting elements (<script> and <template>).

### ISSUE THREAD

**Issue by brandonhundt** [PRX/theworld.org#170]
On landing pages, In the Sidebar Audio list component, Lighthouse is complaining about this affecting a11y. Looks like the list is missing the `<li>` element.

https://web.dev/list/?utm_source=lighthouse&utm_medium=devtools

Where:
![Screen Shot 2021-05-19 at 6.43.25 PM.png](https://images.zenhubusercontent.com/5bd1dc3a4cadef67d2695e38/34efc26b-0b4b-4a6c-9ac2-d07791864c92)
